lib/config/enhance-octokit.js

```diff
@@ -0,0 +1,29 @@
+const OctokitError = require('../models/octokit-error')
+const statsd = require('./statsd')
+
+const instrumentRequests = (octokit, log) => {
+  octokit.hook.wrap('request', async (request, options) => {
+    const requestStart = Date.now()
+    try {
+      const response = await request(options)
+      return response
+    } finally {
+      const elapsed = Date.now() - requestStart
```

Comment:
Can we add a unit test for this to ensure the request timing is getting sent to Datadog? Maybe as a start, we can just make sure `statds.histogram` gets called?

Reply:
Phew, several hours later, we now have [`toHaveSentMetrics`](https://github.com/integrations/jira/pull/268/files#diff-f65ac65e4d6e45df7914e615b009d4c2). This matcher should help us as we add metrics in the future.

I've added a basic test to ensure the histogram is sent as expected.